modules/electrophysiology_browser/php/sessions.class.inc

```diff
@@ -395,10 +395,11 @@ class Sessions extends \NDB_Page
         ];
 
         $queries = [
-            'physiological_electrode'  => 'physiological_electrode_file',
-            'physiological_channel'    => 'physiological_channel_file',
-            'physiological_task_event' => 'physiological_task_event_file',
-            'physiological_archive'    => 'all_files',
+            'physiological_electrode'          => 'physiological_electrode_file',
+            'physiological_channel'            => 'physiological_channel_file',
+            'physiological_task_event'         => 'physiological_task_event_file',
+            'physiological_annotation_archive' => 'physiological_annotation_files',
+            'physiological_archive'            => 'all_files',
```

Comment:
@laemtl @AlexandraLivadas will the `all_files` button download the `annotations` as well? Are they being added afterwards to the archive and then the blake2b recompiled when the annotations are done directly on LORIS and not part of the BIDS dataset right away? Maybe a feature for later.

I'm just thinking that a user would expect to see the annotations files in the downloaded all_files but if the annotations are done through LORIS after the import of the BIDS dataset, then the physiological_archive might need to be updated.

Don't know if I am making any sense, lol. End of the day ;)

Reply:
I get your point and for now, annotations will be part of the zip file if they are provided with the original BIDS files. For the later part, we will probably implement something similar to what you described once annotations will be editable, but it's for a future PR.

Reply:
Perfect! That makes sense. Might be worth at some point to log that into an issue so we do not forget. I tend to forget very easily ;). No rush though.